core/src/main/scala/org/apache/spark/internal/io/SparkHadoopMapReduceWriter.scala

```diff
@@ -119,7 +119,7 @@ object SparkHadoopMapReduceWriter extends Logging {
     }
   }
 
-  /** Write a RDD partition out in a single Spark task. */
+  /** Write an RDD partition out in a single Spark task. */
```

Comment:
It depends from how the acronym is pronounced. For example, 

An MP3 (/ɛm pi θri/)
An RPG (/ɑːr pi ʤi/)
An FBI agent (/ɛf biː aɪ/)